modules/rosa-sts-creating-a-cluster-with-customizations.adoc

```diff
@@ -0,0 +1,224 @@
+// Module included in the following assemblies:
+//
+// * rosa_getting_started_sts/rosa_creating_a_cluster_with_sts/rosa-sts-creating-a-cluster-with-customizations.adoc
+
+[id="rosa-sts-creating-cluster-customizations_{context}"]
+= Creating a cluster with STS using customizations
+
+When you create an OpenShift cluster that uses the AWS Security Token Service (STS), you can customize your installation interactively. When you run `rosa create cluster --interactive` at cluster creation time, you are presented with a series of interactive prompts that enable you to customize your deployment.
```

Comment:
Can we reference that page here? If not via direct link at least tell the users that it exists and the name of the section?